src/services/imu_service/imu_service.cpp

```diff
@@ -98,6 +98,22 @@ void ImuService::OnCreate() {
   ULOG_ARG_INFO(&service_id_, "IMU configured successfully");
 }
 
+bool ImuService::OnStart() {
+  // Validate and parse axis remapping
+  for (int i = 0; i < 3; ++i) {
+    int8_t val = AxisRemap.value[i];
+
+    if (val < -3 || val == 0 || val > 3) {
+      ULOG_ARG_ERROR(&service_id_, "Invalid axis remap value: %d", val);
+      return false;  // FIXME: Should we fail here? Or simply skip and go on with defaul (YardForce) mapping?
```

Comment:
Failing would be the way I think, then the user sees error logs and needs to take action

Reply:
Okay. Wasn't sure (and was to lazy to test) if OnStart() failure would log something which is recognizable by the user.
Just removed the comment.